jep/0000/README.adoc

```diff
@@ -0,0 +1,232 @@
+** The budget is provided according to the rules defined in link:https://developers.google.com/open-source/gsoc/help/org-payments[GSoC Org Payments] (as of 2018 - 2200 USD)
+** Mentor travel grants may cover: transportation costs, visa fees, meals
+** Transportation costs may be covered only for economy-class travel
+** Travel grants do NOT cover travel expenses to any other events
+** Travel grants do NOT cover the hotel costs as long as it covered by the event organizer
+3. Project expenses for students
+** GSoC budget may be used to cover particular expenses for projects: special equipment costs, SaaS hosting
+** Project expenses should happen within the GSoC timeframe between start of Community Bonding and end of the last coding phase
+** GSoC budget may not be used to cover generic equipment costs: laptop, IDEs, internet, etc.
+** GSoC budget may not be used to cover software license costs.
+The most of the projects are expected to be doable with only open-source or freeware software,
+otherwise it is up to the students and mentors to get the setup.
+** Total project expenses may not exceed the amount being payed by Google per project (as of 2018 - 500 USD)
+4. Promotional materials
+** GSoC budget may be used to promote Google Summer of Code by distributing swag, e.g. stickers, T-shirts and other items.
+Swag may be sent to participants or distributed at community events.
+** GSoC budget may be used to cover swag production costs and delivery costs by common mail
+** Swag may be sent out to active GSoC participants, including GSoC students, mentors, org admins, applicants, and other contributors who participated in GSoC
+** GSoC budget may not be used for any kind of payed advertisement or
+payments to external individuals/organizations for promotion.
+** GSoC budget may not be used to send out promotional materials which explicitly or implicitly reference other trademarks or commercial companies
+
+=== Approval process
+
+* All expenses should be pre-approved with
+Jenkins GSoC org admins before the expenses are incurred, and as a pre-condition for paymen
+* Jenkins org admins are eligible to approve the requests only within the GSoC budget defined in this JEP
+* All payments beyond this budget should be approved by Jenkins Governance Meeting or Jenkins Board
+* According to the SPI process,
+the GSoC-related expense reports still need approval by Jenkins Board
+once submitted (see _Payment process_)
+
+=== Payment process
+
+* Jenkins project does not pay for expenses,
+but it reimburses expenses post-factum to students, mentors or Org Admins
+* Reimbursements are performed according to the SPI process defined
+link:https://wiki.jenkins.io/display/JENKINS/Reimbursement+of+Expenses+via+SPI[Reimbursement of Expenses via SPI]
+* It may take a while to get an expense report reimbursed
+(see the _Liability_ section)
+* It is **recommended** that GSoC mentors do the payments for students
+and then reimburse the expenses
```

Comment:
then seek reimbursement for the expenses. Pre-approval is still required in this case.